devicetypes/smartthings/zigbee-battery-accessory-dimmer.src/zigbee-battery-accessory-dimmer.groovy

```diff
@@ -340,7 +340,8 @@ def configure() {
 			zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, BATTERY_VOLTAGE_ATTR, DataType.UINT8, 0, 21600, null)
```

Comment:
Was there a reason that the original DTH didn't use zigbee.batteryConfig? @MMateusiakS

Reply:
cc @tpmanley